solr/solrj/src/java/org/apache/solr/common/util/ContentStreamBase.java

```diff
@@ -47,11 +46,13 @@
 
   public static final String DEFAULT_CHARSET = StandardCharsets.UTF_8.name();
   private static final String TEXT_CSV = "text/csv";
+  public static final String TEXT_XML = "text/xml";
+  public static final String APPLICATION_OCTET_STREAM = "application/octet-stream";
+  public static final String APPLICATION_GZIP = "application/gzip";
+  public static final String APPLICATION_XML = "application/xml";
+  public static final String APPLICATION_JSON = "application/json";
```

Comment:
I'm not sure all these are needed? It looks like https://www.eclipse.org/jetty/javadoc/jetty-10/org/eclipse/jetty/http/MimeTypes.Type.html has replacements for these constants?

Reply:
Errr not sure tying this to Jetty is a good idea. The class is pretty generic it looks like.